_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,308 @@
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol class="list list-md">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ol>
+
+    <ol class="list list-lg">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ol>
+
+    <div class="list list-xl">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Inline List</h3>
+<div class="example-set">
+    <ul class="list list-inline">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with alignments</h3>
+<div class="example-set">
+    <ul class="list">
+        <li class="item align-left">Apples (left aligned)</li>
+        <li class="item align-left">Apricots (left aligned)</li>
+        <li class="item align-left">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (left aligned)
+        </li>
+        <li class="item align-left">Blackberries (left aligned)</li>
+    </ul>
+
+    <ul class="list">
+        <li class="item align-center">Apples (center aligned)</li>
+        <li class="item align-center">Apricots (center aligned)</li>
+        <li class="item align-center">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (center aligned)
+        </li>
+        <li class="item align-center">Blackberries (center aligned)</li>
+    </ul>
+
+    <ul class="list">
+        <li class="item align-right">Apples  (right aligned)</li>
+        <li class="item align-right">Apricots  (right aligned)</li>
+        <li class="item align-right">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+            (right aligned)
+        </li>
+        <li class="item align-right">Blackberries (right aligned)</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Bulleted List</h3>
+<div class="example-set">
+    <ul class="list list-bulleted">
```

Comment:
Maybe add a little margin-left to these content base lists? I find that lists embedded in content are usually well served by being inset a bit.... or ignore me and carry on.